modules/mbedtls/CMakeLists.txt

```diff
@@ -161,6 +161,9 @@ zephyr_interface_library_named(mbedTLS)
 
     zephyr_library_sources(${crypto_source})
 
+    zephyr_library_compile_definitions(MBEDTLSCRYPTO_SOURCE_FILE)
+    zephyr_library_include_directories(internal_include)
```

Comment:
> Even though right now it's only used for the ITS interface fed to Mbed TLS, I purposefully made the naming generic to describe what it actually means in practice.

But as of now (https://github.com/zephyrproject-rtos/zephyr/pull/76222/commits/52da601ccb1a546dbb36e734b4a0ddd6ede10c2c) `MBEDTLSCRYPTO_SOURCE_FILE` is used specifically to set `ITS_CALLER_ID` and not to select which head files to add...
Question: why cannot you just set `zephyr_library_compile_definitions(SECURE_STORAGE_ITS_CALLER_MBEDTLS)` in `zephyr/modules/mbedtls/CMakeLists.txt` and then check in `zephyr/subsys/secure_storage/include/psa/internal_trusted_storage.h` if that symbol is defined or not before assigning `ITS_CALLER_ID`? This avoid defining intermediate symbols and it would also solve the naming point above. Wdyt?

Reply:
Naming changed to `BUILDING_MBEDTLS_CRYPTO` to avoid potential confusion.